eng/pipelines/repoman.yml

```diff
@@ -187,4 +156,52 @@ stages:
               - template: /eng/pipelines/templates/steps/repoman-generate.yml
                 parameters:
                   TargetBranchName: $(TargetBranchName)
-                  RemoteName: azure-samples-main
+                  RemoteName: $(StagingRemoteName)
+
+      - stage: Publish_Repos
+        condition: >-
+          and(
+            succeeded(),
+            or(
+              eq(variables['BuildReasonOverride'], 'Manual'),
+              and(
+                eq('', variables['BuildReasonOverride']),
+                eq(variables['Build.Reason'], 'Manual')
+              )
+            )
+          )
+
+        variables:
+          - template: /eng/pipelines/templates/variables/globals.yml
+
+        jobs:
+        - deployment: Publish_Repos
+          environment: azure-dev
+
+          pool:
+            name: azsdk-pool-mms-ubuntu-2004-general
+            image: azsdk-pool-mms-ubuntu-2004-1espt
```

Comment:
Don't have to worry about it here, but a follow-on to use `image.yml` would be nice.

Reply:
This is a `deployment`